src/server/status_server/mod.rs

```diff
@@ -86,10 +92,11 @@ static FAIL_POINTS_REQUEST_PATH: &str = "/fail";
 pub struct StatusServer {
     thread_pool: ThreadPool,
     tx: Sender<()>,
+    cfg_controller: ConfigController,
     rx: Option<Receiver<()>>,
     addr: Option<SocketAddr>,
+    router: Option<RaftRouter<RocksEngine>>,
```

Comment:
Could we remove `Option` here?

Reply:
It will be hard to write unit tests for status server if `Option` is removed.

Reply:
We can replace `RaftRouter<RocksEngine>` with `T: CasualRouter`, and for unit tests:
```
struct MockRouter;
impl CasualRouter for MockRouter {
   fn send(&self, _: u64, _: CasualMessage) -> Result<()> { Ok(()) }
}
```

Reply:
It's reasonable, I would like to.

Reply:
Field `router` is altered to generic type.